### PR TITLE
Initial irb1600 6 145

### PR DESCRIPTION
**Pull request by gomezc**

### CODE REVIEW

File: abb_irb1600_6_145_support/package.xml [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+      default motion and joint velocity limits.
+    </p>
+    <p>
+      Before using any of the configuration files and / or meshes included
+      in this package, be sure to check they are correct for the particular
+      robot model and configuration you intend to use them with.
+    </p>
+  </description>
+  <author email="aabdelmawgoud@swri.org">Ahmed Abdelmawgoud (Southwest Research Institute)</author>
+  <maintainer email="larmstrong@swri.org">Levi Armstrong (Southwest Research Institute)</maintainer>
```
Comment (130s) [inline abb_irb1600_6_145_support/package.xml]: This maintainer might be outdated, FYI [the latest upstream](https://github.com/ros-industrial/abb_experimental/blob/565ee52000276945eb3cf1c60eaa5cf85dfc1691/abb_experimental/package.xml#L7).

Comment (130s) [reply abb_irb1600_6_145_support/package.xml]: Nevermind my comment for this line. Removing.

**Review by 130s — APPROVED** [review]
(Sorry...I somehow messed my review comments. Re-posting.)

We should be able to backport this to `indigo-devel` as well.

**Review by geoffreychiou — APPROVED** [review]

**Comment by geoffreychiou** [comment]
@shaun-edwards can you please merge, don't have merge rights.

**Pull request merged**